persist/persist_test.go

```diff
@@ -0,0 +1,25 @@
+package persist
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/NebulousLabs/Sia/build"
+)
+
+// TestIntegrationRandomSuffix checks that the random suffix creator creates
+// valid files.
+func TestIntegrationRandomSuffix(t *testing.T) {
```

Comment:
RandomSuffix is an exported function with no state. The definition I've been using is that 'Integration' means no internal state was modified manually and no unexported functions are variables were used.

Perhaps 'Integration' is the wrong word or the definition should be changed, but under that definition this is an integration test.